### PR TITLE
fixed version for static burgers example

### PR DESCRIPTION
**Pull request by ssafaei**

### CODE REVIEW

**Comment by hsorby** [comment]
The order of the Fortran and src directories should be reversed so that we have consistency with other examples.  We should aim to have top level directories src and docs in all the examples and then differentiate the src with Fortran/Python.

**Comment by ssafaei** [comment]
ready to go!

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,4 +4,6 @@ project(Burgers_Static VERSION 1.0 LANGUAGES C Fortran)
 
 find_package(OpenCMISSLibs 1.3.0 REQUIRED COMPONENTS Iron CONFIG)
 
+include(OCMiscFunctions)
```
Comment (hsorby) [inline CMakeLists.txt]: Why did you add this in?  I don't see any of the functions defined in this file being used.

**Comment by hsorby** [comment]
The expected results aren't really going to be the expected output is it?  It doesn't look like something Iron would print out.

**Review by hsorby — COMMENTED** [review]
Clarify why OCMiscFuncitons was added.

**Comment by ssafaei** [comment]
didn't you say it's for testing?

**Comment by ssafaei** [comment]
no, the expected_results is just a text file that existed in the folder.

**Comment by hsorby** [comment]
No not for my testing only for adding a local test, this has nothing do with the functional test framework.  You are safe to remove it.

**Comment by ssafaei** [comment]
done.

**Comment by hsorby** [comment]
Currently the expected results can only be a comparison between output files.  I will add the ability to compare output to stdout but you will have to make sure that the program prints out something really similar to the expected results text.
